templates/internet-of-things/smart-city.html

```diff
@@ -126,533 +136,518 @@ <h1>Build the future with smart cities</h1>
+          clouds. Run it on <a href="https://maas.io/">bare metal</a> or <a href="/cloud">public cloud</a>. From Spark and
+          Hadoop
+          to <a href="/kubeflow">Kubeflow</a>. Better ML. Faster results.
         </p>
+        <div class="p-cta-block">
+          <a href="/ai">Learn more about AI on Ubuntu&nbsp;&rsaquo;</a>
+        </div>
       </div>
-      <div class="col-4">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            {{ image(url="https://assets.ubuntu.com/v1/6e184942-Webinar.svg",
-                        alt="",
-                        width="171",
-                        height="150",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-heading-icon__img"},) | safe
-            }}
-            <h4 class="p-heading-icon__title p-heading--5">Webinar</h4>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>Straightforward compliance</h2>
+        </div>
+        <div class="col">
+          <h3 class="p-heading--5">Open source licensing</h3>
+          <p>
+            Confidently integrate open source components into your proprietary solutions. Benefit from source identification
+            for all software components, all the time. Simplify compliance. Streamline certification. Protect innovation.
+          </p>
+
+          <div class="row">
+            <hr class="p-rule--muted" />
+            <div class="col-3 col-medium-4">
+              <a href="/engage/ubuntu-cybersecurity-webinar">Learn more about Compliance in Ubuntu&nbsp;&rsaquo;</a>
+            </div>
+            <div class="col-9 col-medium-8">
+              <a href="/engage/azure-ubuntu-pro-fips-whitepaper#the-form">Staying compliant while delivering innovation
+                <br class="u-hide--small" />
+                in
+              government services&nbsp;&rsaquo;</a>
+            </div>
           </div>
         </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Let's work together</h2>
+      </div>
+      <div class="col">
         <p>
-          <a href="/engage/open-source-security-cloud-edge-webinar">Securing open source from cloud to edge&nbsp;&rsaquo;</a>
+          We work closely with our customers to solve complex problems together. Our talented engineers are
+          experts in IoT and are well-qualified to provide expertise and guidance to both smaller businesses and larger enterprises.
         </p>
+        <div class="p-cta-block">
+          <a href="#get-in-touch" class="p-button--positive js-invoke-modal">Get in touch</a>
```

Comment:
you should specify a path to a static pages for those that don't have JS enabled. But with JS enabled this should trigger opening a modal form.